sig-big-data/resources.md

```diff
@@ -1,15 +1,61 @@
 # Resources
 
+## Kubernetes Integration status by Big Data product
+
 ### Spark
-* [Spark on Kubernetes Design Proposal](https://docs.google.com/document/d/1_bBzOZ8rKiOSjQg78DXOA3ZBIo_KkDJjqxVuq0yXdew/edit#)
-* [Spark Dynamic Allocation Proposal](https://docs.google.com/document/d/1S9OMnFaeSf_UUxWpMpvC7ERcWx-jDr2g85MWri3Hccc/edit?usp=sharing)
-* [SPARK-JIRA](https://issues.apache.org/jira/browse/SPARK-18278)
-* [Kubernetes Issue #34377](https://github.com/kubernetes/kubernetes/issues/34377)
-* [External Repository](https://github.com/apache-spark-on-k8s/spark)
+
+[Apache Spark](https://spark.apache.org) is a distributed data processing framework. 
+
+##### Status
+
+Kubernetes is supported as a mainline Spark scheduler since [release 2.3](https://spark.apache.org/releases/spark-release-2-3-0.html), see [the detailed documentation](https://spark.apache.org/docs/latest/running-on-kubernetes.html).
+That work was done after the [Spark on Kubernetes original Design Proposal](https://docs.google.com/document/d/1_bBzOZ8rKiOSjQg78DXOA3ZBIo_KkDJjqxVuq0yXdew/edit#)
+in the [apache-spark-on-k8s git repo](https://github.com/apache-spark-on-k8s/spark).
+
+##### Activities 
+
+Enhancements are underway, with a good overview given [in this blog post](https://databricks.com/blog/2018/09/26/whats-new-for-apache-spark-on-kubernetes-in-the-upcoming-apache-spark-2-4-release.html).
+
+* Work is underway for Spark 2.4 to improve support and integration with HDFS.
+  * Design Document: [How Spark on Kubernetes will access Secure HDFS](https://docs.google.com/document/d/1RBnXD9jMDjGonOdKJ2bA1lN4AAV_1RwpU_ewFuCNWKg/edit#heading=h.verdza2f4fyd)
+* shuffle service design
+  * Design Document [Improving Spark Shuffle Reliability](https://docs.google.com/document/d/1uCkzGGVG17oGC6BJ75TpzLAZNorvrAU3FRd2X-rVHSM/edit)
+  * JIRA issue [SPARK-25299: Use remote storage for persisting shuffle data](https://issues.apache.org/jira/browse/SPARK-25299)
 
 ### HDFS
+
+[Apache Hadoop HDFS](https://hadoop.apache.org/hdfs) is a distributed file system, the persistence layer for Hadoop.
+
+##### Status
+
+TODO, e.g. "No release yet."
+
+##### Activities
+
 * [Data Locality Doc](https://docs.google.com/document/d/1TAC6UQDS3M2sin2msFcZ9UBBQFyyz4jFKWw5BM54cQo/edit)
 * [External Repository](https://github.com/apache-spark-on-k8s/kubernetes-HDFS)
```

Comment:
`External Repository` -> `Helm Charts`.

Reply:
clarified the link description, included Helm chart nature of the repo.